lib/modules/manager/devcontainer/extract.spec.ts

```diff
@@ -0,0 +1,363 @@
+import upath from 'upath';
+import * as dockerfileExtract from '../dockerfile/extract';
+import { extractPackageFile } from '.';
+
+function getFixture(content: string) {
+  return {
+    content,
+    path: upath.resolve(__dirname, 'devcontainer.json'),
```

Comment:
```suggestion
    path: 'devcontainer.json',
```

do we really need this simple function? i this this should be inlined